src/main/java/meercode/Lexer/Lexer.java

```diff
@@ -0,0 +1,74 @@
+package meercode.lexer;
+
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public final class Lexer
+{
+    private static String mInCode;
+    private String mFile = "C:\\Users\\franc\\Desktop\\excode.txt";
+    public Lexer()
```

Comment:
Because this is a util class, you should *keep Lexer private* and only use it in a static context.